### PR TITLE
Fb asim 2941 well port

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

File: qmxgraph/page/api.js [hunk 1]
```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
     return model.isVisible(cell);
 };
 
+/**
+ * Sets graph connectable property.
+ *
+ * @param {number} cellId Id of a cell in graph.
+ * @param {boolean} enabled Enable value of this property.
+ */
+graphs.Api.prototype.setConnectable = function setCellsConnectable(cellId, enabled) {
```
Comment (oliveira-esss) [inline qmxgraph/page/api.js]: `setCellsConnectable` is plural but `cellId` must be a single entry. Can we renamed it `setCellConnectable` or work with a list of cell ids?

Comment (prusse-martin) [reply qmxgraph/page/api.js]: The second name is wrong. I will fix it.

File: qmxgraph/page/api.js [hunk 2]
```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
...
+        throw Error("Unable to find cell with id " + cellId);
+    }
+};
+
+/**
+ * Returns current graph connectable property.
+ *
+ * @returns {boolean} Enable value of this property.
+ */
+graphs.Api.prototype.isConnectable = function isCellsConnectable(cellId) {
```
Comment (oliveira-esss) [inline qmxgraph/page/api.js]: This one too should be `isConnectable` I believe.

Comment (prusse-martin) [reply qmxgraph/page/api.js]: 🤦‍♂
Why didn't I check that also?

Comment (oliveira-esss) [reply qmxgraph/page/api.js]: I could have commented in both too.

File: qmxgraph/page/api.js [hunk 3]
```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
...
+ *
+ * @returns {boolean} Enable value of this property.
+ */
+graphs.Api.prototype.isConnectable = function isConnectable(cellId) {
+    "use strict";
+
+    var graph = this._graphEditor.graph;
+    var model = graph.getModel();
+    var cell = this._findCell(model, cellId);
+    return model.isConnectable(cell);
```
Comment (oliveira-esss) [inline qmxgraph/page/api.js]: This checks both cell and graph-wide `connectable` property?

Comment (prusse-martin) [reply qmxgraph/page/api.js]: [isConnectable](https://jgraph.github.io/mxgraph/docs/js-api/files/model/mxGraphModel-js.html#mxGraphModel.isConnectable)

> Returns true if the given mxCell is connectable.  If <edgesConnectable> is false, then this function returns false for all edges else it returns the return value of mxCell.isConnectable.

**Pull request merged**